lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -63,6 +63,7 @@ public class AnsibleModule
             { "selinux_special_fs", null },
             { "shell_executable", null },
             { "socket", null },
+            { "string_conversion_action", null },
```

Comment:
Isn't some code needed here as well?

Reply:
I don't believe so. Since we aren't leveraging this option for Windows modules, we don't want to pass it along.